.github/codecov.yml

```diff
@@ -0,0 +1,7 @@
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 1%
```

Comment:
> `threshold: 1%`

Will there ever be scenarios where the PR has dropped the code coverage by more than `1%` and codecov marks that PR (or commit? sorry, I dont understand the `success` status) a failed?

Reply:
Codecov will not mark the PR as failed, but the codecov report will show the threshold being exceeded. We don't want to mark a PR as failed or blocked due to code coverage, but make it visible enough for the author to pay attention to how much code coverage is being decreased.

Reply:
>  make it visible enough for the author to pay attention to how much code coverage is being decreased

Understood, thanks!